ksp_plugin_adapter/localization/zh-cn.cfg

```diff
@@ -186,7 +188,7 @@ Localization {
     #Principia_FlightPlan_StatusMessage_ChangeFlightPlan = 请更换轨道规划
     #Principia_FlightPlan_StatusMessage_FailedError = 计算失败，出现错误 <<1>> 错误信息 \"<<2>>\"  // <<1>> status_.error <<2>> status_.message
     #Principia_FlightPlan_StatusMessage_TimeOut = 在<<1>>之后  // <<1>> Timeout
-    #Principia_FlightPlan_StatusMessage_MaxSteps = 积分器已达到最大步数<<1>>  // <<1>> Timeout
+    #Principia_FlightPlan_StatusMessage_MaxSteps = 积分器已<<1>>达到最大步数  // <<1>> Timeout
```

Comment:
The existing string talks about the time taken by the computation, not the time reached by the calculation; Zethar recommends:
> 积分器无法在最大步数以内算出+<<1>>时刻